### PR TITLE
[RFC] coverity/108870: Missing break in switch

### PR DESCRIPTION
**Pull request by ghost**
I'm not sure if this is intended...

### CODE REVIEW

File: src/nvim/terminal.c [hunk 1]
```diff
@@ -383,6 +383,7 @@ void terminal_enter(bool process_deferred)
         if (got_bs) {
           goto end;
         }
+        // FALLTHROUGH
```
Comment (splinterofchaos) [inline src/nvim/terminal.c]: Coverity complains about this, but not the above cases? If so, I suppose because they have not statements...

**Comment by splinterofchaos** [comment]
> I'm not sure if this is intended...

My intent was to allow the sending of `<C-n>` to the terminal if not preceded by `<C-\>`. Even were it not for coverity, the `// FALLTHROUGH` definitely makes that more clear. :+1:

**Comment by Pyrohh** [comment]
Thanks @splinterofchaos, merged.

**Comment by justinmk** [comment]
Just FYI, `git merge --no-ff` advice I mentioned was for merges with more than one commit. 1-commit PRs don't need a merge commit, they only need the PR number appended to the commit message.

**Comment by Pyrohh** [comment]
Ok, good to hear, as I've been doing that when merging locally (ever since I started using your script at least). I think I'll read up on this whole merging business, so I can stop using this repo as a training ground :grin:

**Pull request merged**